gulpfile.js

```diff
@@ -27,12 +27,6 @@ const cleanTask = createTask({
   call: task.remove(['build', 'public']),
 })
 
-const lintCssTask = createTask({
```

Comment:
I decided to remove stylelint for the project. It's no longer maintained and was causing some headaches with prettier. And now that we have prettier I feel that is a good replacement, though it would be ideal to add a step in the gulp process to run prettier on our css files to keep things consistent much like stylelint was doing. Let's handle that later